tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -0,0 +1,1164 @@
+import logging
```

Comment:
We will need to add Checkpoint tests for Spark and SQL style Fluent Datasources.